waterloop-site/src/components/RecruitmentForm/Form.tsx

```diff
@@ -0,0 +1,200 @@
+      termTypeCheckedArray: [boolean, boolean, boolean]
+  }
+
+class Form extends React.Component<MyProps, MyState> {
+  constructor(props: MyProps){
+    super(props);
+    this.state = {
+      fullName: "",
+      email: "",
+      program: "",
+      term: "",
+      applicationTerm: "",
+      termType: "",
+      whyJoin: "",
+      technicalAns: "",
+      termCheckedArray: [false, false, false, false, false],
+      termTypeCheckedArray: [false, false, false],
+    };
+  }
+
+  handleChange(event: React.FormEvent<HTMLInputElement>){
+    if (event.currentTarget.name === "entry.1105765972"){ // fullName
+      this.setState({fullName: event.currentTarget.value});
+    }
+    else if (event.currentTarget.name === "entry.204636100"){ // email
+      this.setState({email: event.currentTarget.value});
+    }
+    else if (event.currentTarget.name === "entry.1369449321"){ // program
+      this.setState({program: event.currentTarget.value});
+    }
+    else if (event.currentTarget.name === "entry.1488954434"){ // term
+      this.setState({term: event.currentTarget.value});
+    }
+    else if (event.currentTarget.name === "applicationTerm" && event.currentTarget.id === "0"){
+      this.setState({termCheckedArray: [true, false, false, false, false]});
+      this.setState({applicationTerm: this.props.termList[0]});
+    }
+    else if (event.currentTarget.name === "applicationTerm" && event.currentTarget.id === "1"){
+      this.setState({termCheckedArray: [false, true, false, false, false]});
+      this.setState({applicationTerm: this.props.termList[1]});
+    }
+    else if (event.currentTarget.name === "applicationTerm" && event.currentTarget.id === "2"){
+      this.setState({termCheckedArray: [false, false, true, false, false]});
+      this.setState({applicationTerm: this.props.termList[2]});
+    }
+    else if (event.currentTarget.name === "applicationTerm" && event.currentTarget.id === "3"){
+      this.setState({termCheckedArray: [false, false, false, true, false]});
+      this.setState({applicationTerm: this.props.termList[3]});
+    }
+    else if (event.currentTarget.name === "applicationTerm" && event.currentTarget.id === "4"){
+      this.setState({termCheckedArray: [false, false, false, false, true]});
+      this.setState({applicationTerm: this.props.termList[4]});
+    }
+    else if (event.currentTarget.name === "termType" && event.currentTarget.id === "a"){
+      this.setState({termTypeCheckedArray: [true, false, false]});
+      this.setState({termType: "School Term"});
+    }
+    else if (event.currentTarget.name === "termType" && event.currentTarget.id === "b"){
+      this.setState({termTypeCheckedArray: [false, true, false]});
+      this.setState({termType: "Work/Off Term in Waterloo"});
+    }
+    else if (event.currentTarget.name === "termType" && event.currentTarget.id === "c"){
+      this.setState({termTypeCheckedArray: [false, false, true]});
+      this.setState({termType: "Not in Waterloo"});
+    }
```

Comment:
both `termCheckArray` and `termTypeCheckedArray` only ever appear to have one value that is `true`, and the rest are `false`. These should be simplified into a number field and string field respectively. Let me know if you need some help doing this.